test/e2e/apimachinery/webhook.go

```diff
@@ -66,7 +66,6 @@ const (
 	toBeAttachedPodName       = "to-be-attached-pod"
 	hangingPodName            = "hanging-pod"
 	disallowedConfigMapName   = "disallowed-configmap"
-	nonDeletableConfigmapName = "nondeletable-configmap"
```

Comment:
I expect go fmt will want the rest of the items unindented, run `hack/update-gofmt.sh`

Reply:
ok, I have done this.